guides/govuk-paas.md

```diff
@@ -62,6 +62,119 @@ Instead, transport security is achieved by using HTTPS/TLS and authentication vi
+  * *SpaceDeveloper* for developers. Only for non-production spaces.
+  * *SpaceAuditor* for readonly access
+
+## Support
+
+For users, spaces, organisations ask digital tools in #digital-tools-support Slack channel.
+For technical issues or guidance, the DfE infrastructure engineers can help you. Reach out on #devops-community Slack channel.
+
+The team at GDS can be reached on [cross-government slack](https://ukgovernmentdigital.slack.com/) in the #govuk-paas channel during office hours. They always have one dedicated engineer on support who will answer queries.
+
+The platform is [supported 24x7](https://www.cloud.service.gov.uk/#supported-24-hours-a-day-7-days-a-week) by the PaaS team and they can be reached using the form linked or via email. Support requests are tracked via Zendesk tickets.
+
+In case of suspected outage, always check [the status page](https://status.cloud.service.gov.uk/). It is highly recommended to subscribe to the mailing list (linked on the status page) to learn about potential disruptions.
+
+## Tooling
+
+### cf cli
+
+The default tool to deploy to PaaS is the [cf command line tool](https://docs.cloud.service.gov.uk/get_started.html#set-up-the-cloud-foundry-command-line). It is easy to use and any developer can use it to push applications and create services like postgres or redis.
+
+When deploying with the cli, we recommend using [manifests](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html):
+* It avoids passing multiple options on the command line
+* Manifests can be stored in the git repository
+* Secrets or variables can be passed at deploy time
+
+### Terraform
+The Cloud Foundry provider for Terraform allows describing the deployment in a declarative way. It is the recommended way for robust deployments and continuous delivery pipelines.
+
+#### Why
+
+* The declarative syntax avoids having to think about dependencies. Terraform will create the resources in the right order and destroy them in the right order when required. It also makes it idempotent so terraform won’t recreate a resource already created.
+* This is particularly useful to create disposable environments for testing or review apps, and delete them when they are not required anymore.
+* The syntax is elegant, powerful and modular
+* It can read data from environment variables, AWS parameter store, Azure keyvault, yaml files, json files, etc.
+* It can create an environment made of very different technologies. For example, we create an environment with Azure, PaaS and Statuscake in a single Terraform run.
+
+#### Caution
+
+* Terraform requires to store its state in a persistent storage. An IaaS solution may be used alongside PaaS, like an Azure Storage Account or AWS S3 bucket. Paas also provides an [Amazon S3 service](https://docs.cloud.service.gov.uk/deploying_services/s3/).
+* All the resources are under Terraform control, including databases. Make sure you protect against accidental deletion.
```

Comment:
Is there a recommended way to do this in GOVUK PaaS? Azure had a way of protecting resources from deletion that had to be enabled/disabled outside of ARM templates, for example. If there is a way to do this, it might be worth having a pointer to guidance for how to use it.